tests/cli/test_install_tar.sh

```diff
@@ -0,0 +1,142 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+#####
+#
+# Builds tar image and install it using liveimg kickstart command
```

Comment:
What exactly is this testing? Is this something people do?

Reply:
We don't have accurate information to answer this question but this scenario was identified when we did the original assessment for Composer and has always been part of our integration test suite. So far we've executed it manually.

Reply:
FTR my understanding is that using tarball images to deploy a system is somewhat common, Brian or Terry may know more.

Reply:
Thanks, that helps. I don't see the use-case myself, but I'll investigate.